server/chat-plugins/blackjack.ts

```diff
@@ -398,53 +448,57 @@ class Blackjack extends Rooms.RoomGame {
 			this.deck = new BlackjackDeck().shuffle();
 		}
 
-		player.points = this.getCardPoints(player);
+		player.points = this.getCardPoints(userid);
 
 		if (player.cards.length < 3) return;
 
 		let turnLine = Chat.html`<br /><strong>${player.name}</strong> hit and received [${player.cards[player.cards.length - 1]}] (${player.points})`;
 		this.turnLog += turnLine;
-		if (player.cards.length > 2) this.display(turnLine, null, player.name);
+		if (player.cards.length > 2) this.display(turnLine, false, player.name);
 
-		if (player === this.dealer) {
+		if (player instanceof BlackjackDealer) {
```

Comment:
`instanceof` breaks in hotpatches. I don't see anything wrong with the original if condition, anyway.

Reply:
honestly dont remember makign this change